go.mod

```diff
@@ -10,7 +10,9 @@ require (
 	github.com/go-redis/redis/v8 v8.11.4
 	github.com/gorilla/websocket v1.5.0
 	github.com/grokify/html-strip-tags-go v0.0.1
+	github.com/magiconair/properties v1.8.6
 	github.com/olekukonko/tablewriter v0.0.5
+	github.com/spf13/cast v1.5.0
```

Comment:
Please remove this package as it is not necessary.

Reply:
github.com/spf13/cast has been removed.